.infrastructure/daemonset.yml

```diff
@@ -0,0 +1,21 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: busybox-daemon
+  namespace: mateapp
+spec:
+  selector:
+    matchLabels:
+      app: todoapp-daemon
+  template:
+    metadata:
+      namespace: mateapp
```

Comment:
The namespace 'mateapp' is specified again in the pod template metadata. Ensure consistency with the namespace specified in the metadata section of the DaemonSet.